src/components/button/common/button-common.scss

```diff
@@ -81,6 +82,8 @@ $icon-only: ':where([data-slot-names~=icon], [icon-name]):not([data-slot-names~=
 }
 
 :host([disabled]) {
+  --sbb-button-border-style: dashed;
```

Comment:
Could you please align the variables to the "system" used in button-common: there should be a variable for every state. Moreover, the border color should be smoke (so the current outcome doesn't look correct).